content/topics/Analyze/causal-inference/introduction/synthetic-control-methods-intro.md

```diff
@@ -0,0 +1,207 @@
+- Let $Y_{it}^I$ be the outcome that would be observed for unit $ i $ at time $ t $ if unit $ i $ is exposed to treatment in periods $T_0 + 1$ to $T$.
+- Let $Y_{it}^N$ be the outcome that would be observed for unit $ i $ at time $ t $ in the absence of the intervention. This variable is going to be constructed synthetically. 
+
+We aim to estimate the effect of treatment on the treated unit: 
+
+$
+\tau_{1t} = Y_{1t}^I - Y_{1t}^N = Y_{1t} - Y_{1t}^N
+$
+
+for $t > T_0$, where $Y_{1t}$ is the observed outcome for treated unit. 
+
+In order to create the synthetic control unit, we have to find appropriate weight for each donor in the donor pool by finding: 
+
+- $\mathbf{W}' = (w_2, \ldots, w_{J+1})'$ with $w_j \geq 0$ for $j = 2, \ldots, J + 1$
+- And $w_2 + \cdots + w_{J+1} = 1$. 
+- Each value of $\mathbf{W}'$ represents a potential synthetic control.
+
+The synthetic control has to be as similar as possible to the treatment unit before intervention, hence, the optimal synthetic control units have to be chosen to satisfy the following condition: 
+
+- The vector $\mathbf{W'} = (w_2', \ldots, w_{J+1}')$ is chosen to minimize $||\mathbf{X}_1 - \mathbf{X}_0 \mathbf{W}||$, subject to our weight constraints.
+- Where $\mathbf{X}_1$ be a $(k \times 1)$ vector of pre-intervention characteristics for the treated unit. 
+- Similarly, let $\mathbf{X}_0$ be a $(k \times J)$ matrix of pre-intervention characteristics for the untreated units.
+
+Let $Y_{jt}$ be the value of the outcome for unit $j$ at time $t$. 
+
+For a post-intervention period $t$ (with $t \geq T_0$) the treatment effect with the synthetic control estimator is:
+
+$
+\hat\tau_{1t} = Y_{1t} - \sum_{j=2}^{J+1} w_j^{'} Y_{jt}
+$
+
+Now that we have set a basis for the theoretical aspect of synthetic control methods, let's dive into the implementation in R. 
+
+## Implementation in R 
+
+The implementation steps follow [this](https://github.com/edunford/tidysynth/blob/master/README.md). 
+
+The package we are going to be using is called *tidysynth*, with functionalities specific for the implementation of this method. 
+
+Let's start with installing and loading the package. 
+
+{{% codeblock %}}
+```R
+install.packages('tidysynth')
+library(tidysynth)
+```
+{{% /codeblock %}}
+
+The dataset we are going to use in this tutorial is the *smoking* dataset, based on the paper by Abadie et al. (2010), which evaluates the impact of tobacco control program on cigarette consumption in California.
+
+{{% codeblock %}}
+```R
+require(tidysynth)
+data("smoking")
+View(smoking)
+```
+{{% /codeblock %}}
+
+The method aims to generate a synthetic California using information from a subset of control states (the “donor pool”) where a similar law was not implemented. The donor pool is the subset of case comparisons from which information is borrowed to generate a synthetic version of the treated unit (“California”). 
+
+{{% codeblock %}}
```

Comment:
I'd split the code chunks and take the reader through each step by step:

part 1: synthetic_control function code => explain what is the objective here, which function can be used(i.e. `synthetic_control`) and the options you specify

part 2: generate_predictor => here you are creating predictors from pre-intervention data. Again, motivate why we do this, which function we use and then code snippet. Be specific as to why for eg we take lagged cig sales ( -> to capture historical trends) and not just the code snippet. This will guide readers as to what kind of variables one could potentially use as pre trentment characteristics.

part 3: generate_weights => here we aim to find the optimal weights w_j for each control unit. explain the function and the arguments it takes clearly. 

part 4: generate_control()